src/network/url_downloader.cpp

```diff
@@ -82,7 +82,12 @@ download(QNetworkAccessManager* manager, const Time& timeout, QUrl const& url, P
     QTimer download_timeout;
     download_timeout.setInterval(timeout);
 
-    QNetworkRequest request{url};
+    QUrl adjusted_url{url};
+
+    if (adjusted_url.scheme() == "http")
+        adjusted_url.setScheme("https");
```

Comment:
Now that https is forced, we don't have a test that checks if a url that is http instead of https is properly handled. We'll probably need a new test for this in test_url_downloader.cpp